enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -287,6 +287,41 @@ export default function App() {
 }
 ```
 
+### Embedding the collection browser
+
+With the Collection Browser, you can browse the items in your Metabase instance from your application.
+
+#### Parameters
+
+- **collectionId**: `number` – The numerical ID of the collection. You can find this ID in the URL when accessing a collection in your Metabase instance. For example, the collection ID in `http://localhost:3000/collection/1-my-collection` would be `1`. If no ID is provided, the collection browser will start at the root `Our analytics` collection.
+- **onClick**: `(item: CollectionItem) => void` - An optional click handler that emits the clicked entity.
+- **pageSize**: `number` – The number of items to display per page.
+- **visibleEntityTypes**: `("question" | "model" | "dashboard" | "collection")[]` – the types of entities that should be visible. If not provided, all entities will be shown. 
```

Comment:
The name sounds okay to me as it's generic enough. I feel like it could have been improved although, I don't know a better alternative at the moment.